backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -48,20 +60,26 @@ public UserModel getUserById(
 			.orElseThrow(() -> new ResourceNotFoundException("No user with id=[%s] was found".formatted(id)));
 	}
 
-	//TODO: will use JSON PATCH in later PR
-	@PatchMapping({ "/{id}" })
+	@PatchMapping(path = "/{id}", consumes = "application/json-patch+json")
 	@Operation(summary = "Update a user by ID")
 	@ApiResponse(responseCode = "204", description = "The request has been successfully processed.")
 	public void updateUserById(
 			@NotBlank(message = "id must not be null or blank")
 			@Parameter(description = "The id of the user.", example = "00000000-0000-0000-0000-000000000000")
 			@PathVariable String id,
-			@Validated @RequestBody UserUpdateModel userUpdateModel) {
+			@Validated @RequestBody JsonPatch patch) {
 
 		final var user = userService.getUserById(id)
 				.orElseThrow(() -> new ResourceNotFoundException("No user with id=[%s] was found".formatted(id)));
 
-		userService.updateUser(user.getId(), userUpdateModel.getEmail());
-	}	
+		final var userPatchModel = userPatchModelMapper.toModel(user);
+		final var userPatched = functions.applyJsonPatch(patch, userPatchModel, UserPatchModel.class);
+
+		if (EmailValidator.getInstance().isValid(userPatched.getEmail()) == false) {
+			throw new ResourceNotFoundException("Invalid email=[%s]".formatted(userPatched.getEmail()));
+		}
```

Comment:
Also, `ResourceNotFoundException` isn't the correct error to throw here. This will result in a 404 error, but it *should* be a 400 error.

I sent you a message in Slack about a different way to implement validation, if you decide to go in that new direction, then a 400 response will automatically occur when validation fails.